src/config/keys.py

```diff
@@ -9,6 +9,13 @@
     "AWS_S3_CUSTOM_DOMAIN", "storage.dev.researchhub.com"
 )
 
+AWS_BEDROCK_MODEL_ID = os.environ.get(
+    "AWS_BEDROCK_MODEL_ID", "anthropic.claude-3-5-sonnet-20241022-v2:0"
+)
+AWS_BEDROCK_ANTHROPIC_VERSION = os.environ.get(
+    "AWS_BEDROCK_ANTHROPIC_VERSION", "bedrock-2023-05-31"
+)
```

Comment:
Note: If we don't expect to use different models across different environments, then this is not needed.